app/Http/Resources/OvertimeRequestResource.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use Toby\States\OvertimeRequest\AcceptedByTechnical;
+use Toby\States\OvertimeRequest\Cancelled;
+use Toby\States\OvertimeRequest\Rejected;
+use Toby\States\OvertimeRequest\Settled;
+
+class OvertimeRequestResource extends JsonResource
+{
+    public static $wrap = null;
+
+    public function toArray($request): array
+    {
+        $user = $request->user();
+
+        return [
+            "id" => $this->id,
+            "name" => $this->name,
+            "user" => new SimpleUserResource($this->user),
+            "state" => $this->state,
+            "from" => $this->from->format("Y-m-d H:i"),
```

Comment:
Maybe you could move this very specific format (`""Y-m-d H:i"`) to some helper/constant?

Reply:
added Helper